src/app/types/index.ts

```diff
@@ -15,3 +15,7 @@ export type PoolsCardsProps = {
     assetTokenIcon: string;
   };
 };
+
+export type InputEditedProps = {
+  inputType: "exactIn" | "exactOut";
```

Comment:
"exactIn" | "exactOut" can be an enum rather than a literal enum

Reply:
i am not sure i understand, did you mean i should do something like this below ?
`export type InputEditedProps = {
  inputType: InputEditedType.exactIn | InputEditedType.exactOut;
};`

Reply:
no. U have "exactIn" | "exactOut" that should be enum EnumName {EXACT_IN = "exactIn" | EXACT_OUT = "exactOut"}
inputType: EnumName